docs/tsconfig.json

```
@@ -13,6 +13,5 @@
     "esModuleInterop": true,
     "incremental": true
   },
-  "include": ["next-env.d.ts", "next.config.ts", "types", "src", "pages", "data"],
   "exclude": ["docs/.next", "docs/export", "node_modules"]
```

By the way, shouldn't it be:

```suggestion
  "exclude": [".next", "export", "node_modules"]
```